redash/query_runner/big_query.py

```diff
@@ -207,7 +221,7 @@ def _get_query_result(self, jobs, query):
 
         rows = []
 
-        while ("rows" in query_reply) and current_row < query_reply['totalRows']:
+        while ("rows" in query_reply) and int(current_row) < int(query_reply['totalRows']):
```

Comment:
`current_row` / `totalRows` aren't integers?

Reply:
yaa .. 
it was not integers before , cz this kind of errors always happened when try to connecting to bigquery
`TypeError: '<' not supported between instances of 'str' and 'int'`
 , even all the schema already shows at query page

Reply:
This was fixed in #4481.